src/main/java/com/wow/server/data/model/User.java

```diff
@@ -11,9 +14,11 @@
 
 @Entity
 @Table(name = "TB_USER")
+@Getter
+@Setter
 public class User {
 
-	@Id @GeneratedValue(strategy = GenerationType.AUTO)
+	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Had to change to Identity as AUTO was not taking into account already created entries

Will need to take closer look when schema auto creation will be worked on